GitUI/UserControls/RevisionGrid/RevisionGridControl.cs

```diff
@@ -671,6 +673,11 @@ public bool SetSelectedRevision(ObjectId? objectId, bool toggleSelection = false
                 return true;
             }
 
+            if (showNoRevisionMsg)
```

Comment:
Why would we not want to show the message?

Reply:
If the check is done in an "automatic" flow where you do not realize what went wrong.
Also if you for some reason manage to check a complete ObjectId that is not in the repo at all.

Both of these arguments (especially the second one) is possibly not valid scenarios, but I cannot verify all scenarios so this is taking it safe.